src/main/resources/jenkins/branch/OrganizationFolder/configure-entries.jelly

```diff
@@ -24,10 +24,12 @@
  -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
-    <f:entry field="navigators" title="Repository Sources">
-        <f:repeatableHeteroProperty field="navigators" hasHeader="true"/>
-    </f:entry>
-    <f:entry field="projectFactories" title="Project Recognizers">
-        <f:repeatableHeteroProperty field="projectFactories" hasHeader="true"/>
-    </f:entry>
+    <f:section title="${%Project Sources}">
+        <f:entry field="navigators" title="${%Repository Sources}">
```

Comment:
[2 space indent in XML](https://github.com/jenkinsci/branch-api-plugin/blob/master/CONTRIBUTING.md#indentation)